swarm/runtime/statsdb/ingestion.py

```
@@ -49,6 +50,73 @@ def _insert_raw_event(self, event: Dict[str, Any]) -> bool:
+                    SELECT event_id, seq, run_id, ts, kind, flow_key, step_id, agent_key, payload
+                    FROM {temp_table}
+                    ON CONFLICT (event_id) DO NOTHING
+                    RETURNING event_id
+                """
+                ).fetchall()
+
+                conn.execute(f"DROP TABLE {temp_table}")
+
+                return set(r[0] for r in result)
+
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)